### PR TITLE
Update android export page for AAB requirement

### LINKED ISSUE #9134: Exporting for Android (Exporting for Google Play Store) should reference AAB requirement
**Your Godot version:**

4.2

**Issue description:**

The section on exporting for the Google Play store still refers to creating a APK, but Google Play no longer accepts APK for new projects, it must be an AAB.  That requires using the Gradle build and template, it also requires Java 17 to successfully build or you get odd error messages.  I was using Open JDK 11, and if it wasn't for some offhand comment in a posting somewhere I never would have known I needed to get JDK 17.

**URL to the documentation page:**

https://docs.godotengine.org/en/stable/tutorials/export/exporting_for_android.html#exporting-for-google-play-store

I believe fixing this requires updating this section to address the new Google Play requirements (specifically AAB) and the consequences.  It would be good to have some pointers about the errors that can be ignored or followed up on.

### PR DESCRIPTION
**Pull request by skyace65**
Does what the title says. Since august of 2021 all new apps uploaded to Google Play must be in AAB format, that requirement is noted on the top of [this page](https://developer.android.com/guide/app-bundle/) in the Android documentation.

I've added a sentence on that requirement and re-worded a few things. I've also removed the section on multiple APK signing, that's obsolete with the AAB format. And I figure anyone still uploading APKs because their app was grandfathered in should know how to do that already if they don't want to switch to AABs for some reason.

Closes #9134.

### CODE REVIEW

File: tutorials/export/exporting_for_android.rst [hunk 1]
```diff
@@ -137,23 +140,16 @@ Don't forget to uncheck the **Export With Debug** checkbox while exporting.
...
 
-By default, the APK will contain native libraries for both ARMv7 and ARMv8
-architectures. This increases its size significantly. To create a smaller APK,
+By default, the AAB or APK will contain native libraries for both ARMv7 and ARMv8
+architectures. This increases its size significantly. To create a smaller file,
 uncheck either **Armeabi-v 7a** or **Arm 64 -v 8a** in your project's Android
-export preset. This will create an APK that only contains a library for
+export preset. This will create an file that only contains a library for
 a single architecture. Note that applications targeting ARMv7 can also run on
 ARMv8 devices, but the opposite is not true.
```
Comment (Calinou) [inline tutorials/export/exporting_for_android.rst:151]: With AAB, the practice of including less architectures to reduce download size is obsolete. Google automatically splits up the AAB on their backend so the user only downloads the architecture they need.

**Comment by skyace65** [comment]
I've updated that section to state that you don't need to make multiple AABs, and added an explanation at the end of the paragraph.

File: tutorials/export/exporting_for_android.rst [hunk 2]
```diff
@@ -137,22 +140,17 @@ Don't forget to uncheck the **Export With Debug** checkbox while exporting.
...
-Since August 2019, Google Play requires all applications to be available in
-64-bit form. This means you cannot upload an APK that contains *just* an ARMv7
-library. To solve this, you can upload several APKs to Google Play using its
-`Multiple APK support <https://developer.android.com/google/play/publishing/multiple-apks>`__.
-Each APK should target a single architecture; creating an APK for ARMv7
-and ARMv8 is usually sufficient to cover most devices in use today.
+Optimizing the file size
+------------------------
+
+If you're working with APKs an not AABs, By default, the APK will contain native
```
Comment (mhilbrunner) [inline tutorials/export/exporting_for_android.rst:146]:
```suggestion
If you're working with APKs and not AABs, by default, the APK will contain native
```

File: tutorials/export/exporting_for_android.rst [hunk 3]
```diff
@@ -137,22 +140,17 @@ Don't forget to uncheck the **Export With Debug** checkbox while exporting.
...
+libraries for both ARMv7 and ARMv8 architectures. This increases its size significantly.
+To create a smaller file, uncheck either **Armeabi-v 7a** or **Arm 64 -v 8a** in
+your project's Android export preset. This will create an APK that only contains
+a library for a single architecture. Note that applications targeting ARMv7 can
+also run on ARMv8 devices, but the opposite is not true. The reason you don't do
+this to save space with AABs is that google automatically splits up the AAB on their
```
Comment (mhilbrunner) [inline tutorials/export/exporting_for_android.rst:152]:
```suggestion
this to save space with AABs is that Google automatically splits up the AAB on their
```

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thanks!

**Comment by mhilbrunner** [comment]
Cherrypicked to 4.3 in https://github.com/godotengine/godot-docs/pull/10347.

**Pull request merged**